lms/templates/courseware/courseware.html

```diff
@@ -48,7 +48,7 @@
 
 </%block>
 
-<%block name="headextra">
+<%block name="headextra_css">
```

Comment:
Nit: would you mind indenting the contents inside the block so that it is clearer where it ends.